docker/README.md

```diff
@@ -62,6 +71,14 @@ From DSpace/DSpace-angular
 docker-compose -p d7 -f docker/docker-compose.yml up -d
 ```
 
+## Run DSpace Angular dist build with DSpace Demo site backend
+
+```
+docker compose -f docker/docker-compose-dist.yml pull  # get most recent build
+docker compose -f docker/docker-compose-dist.yml build # or, using code in your branch
+docker compose -f docker/docker-compose-dist.yml up -d # start the server
```

Comment:
I'd recommend aligning these with the other instructions in this file (just for consistency).  These look more like the other commands in this README and I think they are equivalent.
```
docker-compose -f docker/docker-compose-dist.yml pull
docker-compose -f docker/docker-compose-dist.yml build
docker-compose -p d7 -f docker/docker-compose-dist.yml up -d
```